plugins/module_utils/network/vyos/argspec/ospf_interfaces/ospf_interfaces.py

```diff
@@ -76,7 +76,7 @@ def __init__(self, **kwargs):
                                     "type": "dict",
                                     "options": {
                                         "key_id": {"type": "int"},
-                                        "key": {"type": "str"},
+                                        "key": {"type": "str", "no_log": True},
```

Comment:
Same as above. In fact, for older modules, this might be a breaking change for existing users.